README.rst

```diff
@@ -41,6 +41,14 @@ the server::
 For available options check
 https://github.com/TeamHG-Memex/arachnado/blob/master/arachnado/config/defaults.conf.
 
+Test
+-----------
+To start unit tests for API:
+
+python -m tornado.test.runtests tests.test_data
+or
+python3 -m tornado.test.runtests tests.test_data
```

Comment:
This command won't run doctests; I wonder if it is possible to use a common test runner to run them all (pytest?).